src/app/Question.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+
+class Question extends Model
+{
+    use SoftDeletes;
+    
+    protected $guarded = array('id');
+
+    public static $rules = array(
+        'big_question_id' => 'required',
+        'place_name' => 'required'
+    );
+
+    public function choice()
```

Comment:
複数選択肢があるのであればchoicesではないかと思います。
```suggestion
    public function choices()
```